packages/cubejs-questdb-driver/test/QuestQuery.test.ts

```diff
@@ -0,0 +1,172 @@
+import { QuestQuery } from '../src/QuestQuery';
+import { prepareCompiler as originalPrepareCompiler } from '@cubejs-backend/schema-compiler';
+
+const prepareCompiler = (content: string, options: any[]) => originalPrepareCompiler({
+  localPath: () => __dirname,
+  dataSchemaFiles: () => Promise.resolve([
+    { fileName: 'main.js', content }
+  ])
+}, { adapter: 'postgres', ...options });
+
+describe('QuestQuery', () => {
+  const { compiler, joinGraph, cubeEvaluator } = prepareCompiler(`
+    cube(\`visitors\`, {
+      sql: \`
+      select * from visitors
+      \`,
+
+      measures: {
+        count: {
+          type: 'count'
+        },
+
+        unboundedCount: {
+          type: 'count',
+          rollingWindow: {
+            trailing: 'unbounded'
+          }
+        }
+      },
+
+      dimensions: {
+        createdAt: {
+          type: 'time',
+          sql: 'created_at'
+        },
+        name: {
+          type: 'string',
+          sql: 'name'
+        }
+      }
+    })
+
+    cube(\`Deals\`, {
+      sql: \`select * from deals\`,
+    
+      measures: {
+        amount: {
+          sql: \`amount\`,
+          type: \`sum\`
+        }
+      },
+
+      dimensions: {
+        salesManagerId: {
+          sql: \`sales_manager_id\`,
+          type: 'string',
+          primaryKey: true
+        }
+      }
+    })
+    
+    cube(\`SalesManagers\`, {
+      sql: \`select * from sales_managers\`,
+    
+      joins: {
+        Deals: {
+          relationship: \`hasMany\`,
+          sql: \`\${SalesManagers}.id = \${Deals}.sales_manager_id\`
+        }
+      },
+      
+      measures: {
+        averageDealAmount: {
+          sql: \`\${dealsAmount}\`,
+          type: \`avg\`
+        }
+      },
+    
+      dimensions: {
+        id: {
+          sql: \`id\`,
+          type: \`string\`,
+          primaryKey: true
+        },
+    
+        dealsAmount: {
+          sql: \`\${Deals.amount}\`,
+          type: \`number\`,
+          subQuery: true
+        }
+      }
+    });
+    `, []);
+
+  it('test equal filters', async () => {
```

Comment:
We don't need this test here, just the ORDER BY / GROUP BY.

Reply:
I'd prefer to keep this test since it validates that `= NULL` filter expression is being used instead of `IS NULL`. So, it's as valuable as the subsequent ORDER BY / GROUP BY tests.